src/bootstrap/src/core/config/tests.rs

```diff
@@ -39,7 +37,11 @@ fn prepare_test_specific_dir() -> PathBuf {
     // Replace "::" with "_" to make it safe for directory names on Windows systems
     let test_path = current.name().unwrap().replace("::", "_");
 
-    let testdir = parse("").tempdir().join(test_path);
```

Comment:
Here the point was to use the `tempdir` of the normal bootstrap build directory, rather than something in `/tmp`. If we switch to `TestCtx`, there's no point in doing that anymore, and we can just use `tempfile` here directly, without creating a config.